### PR TITLE
[FIX] Test for The E.N.G.I.N.E.

### PR DESCRIPTION
**Pull request by mgolawski**
[FIX](https://digitalrig.atlassian.net/browse/DR-XX) Test for The E.N.G.I.N.E.

## Checklist:
- [x] Fix Cypress e2e tests
- [x] Add tests for quotesService
- [x] Add tests for path services
- [x] Guard path services for edge cases

### CODE REVIEW

File: src/utils/__tests__/quotesService.test.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import { FRONT_END_ROLE, BACK_END_ROLE, DEV_OPS_ROLE } from '../../constants/roles';
+import { BRAG, CONFESS } from '../../constants/roleActions';
+
+import { getQuote, getRandomQuoteID } from '../quotesService';
+
+jest.mock('../../quotes/backEndQuotes.js', () => ({
```
Comment (blurbyte) [inline src/utils/__tests__/quotesService.test.js]: Maybe it would be good idea to actually keep these functions pure and pass data as last argument?

File: src/utils/__tests__/quotesService.test.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    expect(quote).toEqual("Daily in 5 minutes and I'm still not sure what to say...");
+  });
+});
+
+describe('quotesService - getRandomQuoteID', () => {
+  it('should return random possible quoteID for given action and role', () => {
+    const randomQuoteID = getRandomQuoteID(BACK_END_ROLE, BRAG);
+
+    expect(['1', '2']).toContain(randomQuoteID);
```
Comment (blurbyte) [inline src/utils/__tests__/quotesService.test.js]: I guess there are no better way to do it?

File: src/utils/extractActionFromPath.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 function extractActionFromPath(path) {
-  return path.split('/')[2];
+  return path.split('/')[2] || '';
```
Comment (blurbyte) [inline src/utils/extractActionFromPath.js]: Is it really good idea? Returning undefined if it doesn't exist looks like good deal for me.

File: src/utils/__tests__/quotesService.test.js (hunk 1 above)
Comment (mgolawski) [reply src/utils/__tests__/quotesService.test.js]: I don't get it. Can you elaborate?

File: src/utils/__tests__/quotesService.test.js (hunk 2 above)
Comment (mgolawski) [reply src/utils/__tests__/quotesService.test.js]: The best what i found. Looks simple anyway so i would stick to it.

File: src/utils/__tests__/quotesService.test.js (hunk 1 above)
Comment (blurbyte) [reply src/utils/__tests__/quotesService.test.js]: `function getQuote(role, action, quoteID, quotes)` - that way no need to mock deps
